src/lib.rs

```diff
@@ -0,0 +1,23 @@
+#![no_std]
+
+mod addr;
+mod config;
+mod frame;
+mod page_table;
+mod page_table_entry;
+mod pte_flags;
+
+extern crate alloc;
+
+pub use crate::page_table_entry::PageTableEntry;
+pub use crate::pte_flags::PTEFlags;
+pub use addr::*;
+pub use config::*;
+pub use frame::*;
+pub use page_table::*;
+
+pub fn init_frame_allocator(from: usize, to: usize) {
```

Comment:
it's called here: https://github.com/binary-bruce/extract-page-table/blob/main/os/src/mm/mod.rs#L23